### ISSUE TITLE
Video playback support

### ISSUE THREAD

**Issue by rs189** [turanszkij/WickedEngine#169]
As the name implies, it would be nice to have the option to play videos, maybe take a look into - https://ffmpeg.org/

**Comment by turanszkij** [comment]
Agreed, it would be useful. I'm not sure when I'll get around to do it.

**Comment by turanszkij** [comment]
This is being handled here: #661
